04/app.js

```diff
@@ -0,0 +1,31 @@
+const divsList = document.querySelectorAll('div');
+const bodyEl = document.querySelector('body');
+
+const addClassClicked = function (e) {
+    if (this.className === 'child' || 'parent') {
```

Comment:
Jeśli chcemy iść w tą stronę to musimy zrobić tak: `this.className === 'child' || this.className === 'parent'`, w twoim przypadku `'parent'` to po prostu wartość prawdziwa.